server/tests/testarea.py

```diff
@@ -0,0 +1,39 @@
+from gqlComms import listHistograms, createHistogram, deleteHistogram,getHistogram,getHistograms
+import numpy as np
+import datetime
+import argparse
+
+def test_createHist():
+    NUM = 100
+    LENGTH = 50
+    LOW = 0
+    HIGH =1000
```

Comment:
Please refer to PR #33 and update your precommit script, as we now have python black to help with formatting